notion_haystack/notion_exporter.py

```diff
@@ -38,13 +37,14 @@ def __init__(
             exclude_title_containing=exclude_title_containing,
         )
 
-    def run(self, file_paths: List[str])-> Tuple[Dict, str]:
+    @component.output_types(documents=List[Document])
+    def run(self, page_ids: List[str])-> Tuple[Dict, str]:
```

Comment:
The output type annotation doesn't apply anymore for Haystack 2.0
```suggestion
    def run(self, page_ids: List[str]):
```